service/src/main/java/org/databiosphere/workspacedataservice/service/NoCacheFilter.java

```diff
@@ -0,0 +1,31 @@
+package org.databiosphere.workspacedataservice.service;
+
+import jakarta.servlet.*;
```

Comment:
super nit: get rid of the wildcard import
```suggestion
import jakarta.servlet.FilterChain;
import jakarta.servlet.ServletException;
```